client/src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { 
+  IonContent,
+  IonCard, 
+  IonCardHeader,
+  IonCardTitle,
+  IonCardSubtitle,
+  IonCardContent,
+  IonItemGroup,
+  IonItemDivider,
+  IonList,
+  IonItem,
+  IonLabel
+} from '@ionic/react';
+import { Header } from '../components';
+import { useKeycloak } from '@react-keycloak/web';
+import { AppContext } from '../AppContext';
+import { RouteComponentProps } from 'react-router';
+
+const userInit = {
+  username: 'unknown',
+  email: 'unknown',
+  firstName: 'unknown',
+  lastName: 'unknown',
+  emailVerified: false,
+}
+
+export const ProfilePage: React.FC<RouteComponentProps> = ({ match }) => {
+  const { keycloakEnabled } = useContext(AppContext);
+  const [keycloak] = useKeycloak();
+  const [user, setUser] = useState<Keycloak.KeycloakProfile>(userInit);
```

Comment:
From all 3 statements we can use just fact that we have user and token hooked into some variable somewhere. 
Having user carries a lot of information:

- That keycloak is enabled
- That we logged in successfuly
- That we have active token to perform request (can be still invalid and we will get redirect from server)
-